test/DrawBBox.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python2
+import subprocess 
+import numpy
+import sys
+
+# Edit this as required (windows, mac).  Be able to write this.
+PLY_FILE="/tmp/displaz_bbox.ply"
+
+# write .ply file representing a box with specfied center and transform to f
+def MakeBBox(v_center, transform, f):
+  f.write("ply\n")
+  f.write("format ascii 1.0\n")
+  f.write("comment simple rectangular prism\n")
+  f.write("element vertex 8\n")
+  f.write("property float x\n")
+  f.write("property float y\n")
+  f.write("property float z\n")
+  f.write("element face 2\n")
+  f.write("property list uint int vertex_index\n")
+  f.write("end_header\n")
+
+  # apply transform to unit vectors and then add result to center
+  unit_vectors = [ 
+    [0, 1, -1, 0], [1, 0, -1, 0], [0, -1, -1, 0], [-1, 0, -1, 0], 
+    [0, 1, 1, 0], [1, 0, 1, 0], [0, -1, 1, 0], [-1, 0, 1, 0]
+  ] 
+  xfm_unit_vectors = [transform.dot(unit) for unit in unit_vectors]
+  vertices = [numpy.add(v_center, xuv) for xuv in xfm_unit_vectors]
+
+  # write out resulting vertices
+  map(lambda v: f.write("%s\n" % " ".join(str(elem) for elem in v)), vertices)
```

Comment:
As a small nicety, this might be somewhat clearer as a simple `for` loop (`map` would normally be used to gather output, but here it's being used more as a `foreach`, which is something python lacks):

```python
v_center = np.array(v_center)
for uvec in unit_vectors:
    vertex = v_center + transform.dot(uvec)
    f.write("%.17g %.17g %.17g\n" % tuple(vertex))
```